.golangci.yml

```diff
@@ -1,13 +1,5 @@
 run:
   timeout: 10m
-  # Enable checking the by default skipped "examples" dirs
-  skip-dirs:
-    - vendor$
-    - third_party$
-    - testdata$
-    - Godeps$
-    - builtin$
-  skip-dirs-use-default: false
```

Comment:
Why this change?  I assume this was there for a reason.

Reply:
`.golangci.yml` is one of the files managed by ci-mgmt for better consistency in style and linting across projects. As part of onboarding to ci-mgmt it's going to continue managing the file, although we rarely (if ever) make changes here because it usually means we need to update failures all over the place.

I don't love solutions like this that don't allow for some customization (I usually want more aggressive linters than what we have here), but consistency and predictability are always more important than personal preferences.

IMO mise handles this much better. ci-mgmt ships a default configuration under `.config/mise.toml` which can be overridden with a top-level `mise.toml`. This way something like p-kubernetes can declare it needs kubectl on top of all the usual tools.